.github/workflows/testpypi.yml

```diff
@@ -0,0 +1,40 @@
+# Publish python package at https://test.pypi.org/p/ceph-nvmeof
+# This is for dev releases, and will be pushed to test pypi on all merges to "devel" branch.
+# Usage: `pip install --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple ceph-nvmeof==1.2.15`
+name: Upload release to Test PyPI
+on:
+  push:
+    tags:
+      - '*'
+    branches:
+      - 'devel'
```

Comment:
Given that the test and prod workflows are 99% same, I'd try to merge them into a single one accepting an input flag (`prod`/`test`), or building `prod` packages for `v*.*.*` tags, and `test` otherwise. From experience, maintaining 2 different versions of the (almost) same workflow is counterproductive and leads to mismatches.